docker-compose.yml

```diff
@@ -1,31 +1,45 @@
-version: "3"
+version: '3.3'
+
 services:
-  api:
+
+  database:
     image: mysql
-    # NOTE: use of "mysql_native_password" is not recommended: https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password
-    # (this is just an example, not intended to be a production configuration)
-    command: --default-authentication-plugin=mysql_native_password
     restart: always
+    command: --default-authentication-plugin=mysql_native_password
+    ports:
+      - "3306:3306"
+    volumes:
+      - ./.database/data:/var/lib/mysql
+      - ./.database/log:/var/log/mysql
+      - ./.database/mysql:/etc/mysql/conf.d
+      - ./database/exip.sql:/docker-entrypoint-initdb.d/0_database.sql
+    environment:
+      MYSQL_DATABASE:
+      MYSQL_ROOT_PASSWORD:
+
+  api:
     build: ./src/api
+    image: exip/src/api
+    restart: always
+    command: npm run start
+    depends_on:
+      - database
     ports:
-      - "5005:5005"
+      - "5001:5001"
     volumes:
-      - ./api/src:/app/src:ro
-      # - ./src/api/.keystone:/app/src/.keystone:ro
-      # - ./src/api/:/app/src/:ro
-      - ./api/src/.keystone:/app/src/.keystone:ro
+      - ./.api/src:/app/src:ro
+      - ./.api/src/.keystone:/app/src/.keystone:ro
     environment:
-      - MYSQL_ROOT_PASSWORD=example
-      - PORT=5005
-      - DB_URI=mysql://root@localhost:5005/exip
-      - SESSION_SECRET=asdfaslfdjjasjoiwefjwoerij23j8j2oi2j
-      - GOV_NOTIFY_API_KEY
-    entrypoint: npx nodemon src/keystone.ts -L
+      PORT: 5001
+      DATABASE_URL:
+      GOV_NOTIFY_API_KEY:
+
```

Comment:
we also need to add COMPANIES_HOUSE_API_URL and COMPANIES_HOUSE_API_KEY

Reply:
Added.